webapp/i18n/de.json

```diff
@@ -1116,7 +1116,7 @@
   "create_team.display_name.required": "Dieses Feld ist erforderlich",
   "create_team.display_name.teamName": "Teamname",
   "create_team.team_url.back": "Zurück zum vorigen Schritt",
-  "create_team.team_url.charLength": "Der Name muss mindestens 2 Zeichen und darf maximal 15 Zeichen lang sein",
+  "create_team.team_url.charLength": "Der Name muss mindestens {min} Zeichen und darf maximal {max} Zeichen lang sein",
```

Comment:
Gotcha, will back these changes out. It's a little concerning that some files have a min of '2' and some have it as '4' ...